Update troubleshooting makefile (#12)
logzio/logzio-pubsub: README.md

##### 2. Build your credentials file

Create a working directory for this step and `cd` into it, run as root.

```shell

> mkdir ~/logzio-pubsub && cd ~/logzio-pubsub

Reviewer: Support has asked that we avoid relative paths.

```

You'll need to build a credentials file so Pub/Sub can authenticate
and get the right permissions.

You can build it through:
* [The command line](#credentials-cmd) 
* [The Cloud console](#credentials-console)

<div id ="credentials-cmd">

###### Option 1: In the command line

Build your credentials file using your Google Cloud project ID.  

> Before you begin make sure your 'gcloud' cli is installed. If not, execute the following:

Reviewer: ```suggestion
Before you begin, make sure your 'gcloud' CLI is installed. If not, execute the following:
```

```yaml
listener: <<LISTENER-HOST>>
pubsubs:
 - project_id: PROJECT-1_ID
   topic_id: TOPIC-1_ID
   token: <<SHIPPING-TOKEN>>
   subscriptions: [SUB1_ID, SUB2_ID, SUB3_ID]
   type: stackdriver

 - project_id: PROJECT-1_ID
   topic_id: TOPIC-2_ID
   token: <<SHIPPING-TOKEN>>
   subscriptions: [SUB1_ID, SUB2_ID, SUB3_ID]
   type: stackdriver

 - project_id: PROJECT-3_ID
   topic_id: TOPIC-1_ID
   token: <<SHIPPING-TOKEN>>
   subscriptions: [SUB1_ID, SUB2_ID, SUB3_ID]
   type: stackdriver
```
** Note that YAML files are sensitive to spaces and tabs. We recommend using a YAML validator to make sure that the file structure is correct.

Click here for more information about [filebeat for Google Cloud Pub/Sub](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-input-google-pubsub.html#filebeat-input-google-pubsub).
###### Configuration instructions

| Parameter | Description |
|---|---|
| listener | The Logz.io listener host. (Default value: `listener.logz.io`) <br> Replace `<<LISTENER-HOST>>` with your region's listener host. For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html). |

> | pubsubs | This is an array of one or more GCP subscriptions. For each topic, provide topic and subscriptions IDs, as given from Pub/Sub. |
> | token | Your Logz.io shipping token. For each project under `pubsubs`. <br> Replace `<<SHIPPING-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to. You can send your logs to different accounts that are in the same region, you can do that by inserting different tokens. |
> | credentials_file (Not required, Default value: '<project_id>-credentials.json') | This field is only required if your credentials file is named differently than the default value. For an example of adding this field go to [input example file](https://github.com/logzio/logzio-pubsub/blob/master/pubsub-input-example.yml). |

Reviewer: ```suggestion
| pubsubs | This is an array of one or more GCP subscriptions. For each topic, provide topic and subscription IDs, as given from Pub/Sub. |
| token | Your Logz.io shipping token. For each project under `pubsubs`. <br> Replace `<<SHIPPING-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to. You can send your logs to different accounts that are in the same region, you can do that by inserting different tokens. |
| credentials_file (Not required, Default value: '<project_id>-credentials.json') | This field is only required if your credentials file was renamed and does not have the default name.  For an example of adding this field go to [input example file](https://github.com/logzio/logzio-pubsub/blob/master/pubsub-input-example.yml). |
```
